doc/source/contrib/contributing.rst

```diff
@@ -295,12 +295,12 @@ Browser requests
 Once your application has started as describe above,
 you can try your oereb server as follows.
 In your browser, check the following URL:
-   * http://localhost:6543/oereb/versions
-   * http://localhost:6543/oereb/capabilities
+* http://localhost:6543/oereb/versions
+* http://localhost:6543/oereb/capabilities
```

Comment:
The official path are with the fomat (json or xml):
* http://localhost:6543/oereb/versions/json
* http://localhost:6543/oereb/capabilities/json
http://localhost:6543/oereb/getegrid/json?XY=2608883,1261844

Reply:
Indeed, thanks